source/jesp.adoc

```diff
@@ -15,6 +15,7 @@ The Jakarta EE Specification Committee hereby adopts the https://www.eclipse.org
 .. Service Release Review: 14 calendar days; and
 .. JESP Update: 7 calendar days.
 
+. Any specification committee approval ballot may be closed after 7 calendar days if all working group members or member representatives, elligible to vote, have cast their vote. Otherwise, all ballot durations shall be as listed above.
```

Comment:
```suggestion
. Any specification committee approval ballot may be closed after 7 calendar days if all working group members or member representatives, eligible to vote, have cast their vote. Otherwise, all ballot durations shall be as listed above.
```